idd/Energy+.idd.in

```diff
@@ -22292,12 +22292,16 @@ AirflowNetwork:MultiZone:Zone,
       \note Selecting Advanced results in EnergyPlus calculating modified Wind Pressure Coefficients
       \note to account for wind direction and turbulence effects on single sided ventilation rates.
       \note Model is only valid for zones with 2 openings, both of which are on a single facade.
-  N6; \field Facade Width
+  N6, \field Facade Width
       \type real
       \units m
       \minimum 0.0
       \default 10.0
       \note This is the whole building width along the direction of the facade of this zone.
+  A6; \field Occupant Ventilation Control Name
+      \type object-list
+      \object-list AirflowNetwork:OccupantVentilationControlNames
```

Comment:
@lgu1234 @jasondegraw I would delete the colon from the object-list reference.  Not sure if it would cause problems, but we don't normally use them here.  Change all occurences of this.